test/check-machines-consoles

```diff
@@ -35,15 +36,15 @@ class TestMachinesConsoles(VirtualMachinesCase):
         b.wait_in_text("body", "Virtual machines")
         self.waitVmRow("subVmTest1")
 
-        b.wait_in_text("#vm-subVmTest1-state", "Running") # running or paused
+        b.wait_in_text("#vm-subVmTest1-state", "Running")  # running or paused
```

Comment:
This "autopep8" commit should go away after a re-import after https://github.com/cockpit-project/cockpit/pull/15561 lands